src/Ranking.elm

```diff
@@ -1,17 +1,140 @@
 module Ranking exposing (Model, Msg, init, update, view)
 
-import Debug exposing (todo)
-import Zipper exposing (Zipper)
+import Browser
+import Colors exposing (black, blue, blueGreen, lightBlue, orange, purple, sky, white)
+import Element exposing (alignLeft, alignRight, centerX, centerY, column, el, fill, height, html, layout, maximum, padding, rgb255, shrink, spacing, table, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font exposing (center)
+import Element.Input as Input exposing (button)
+import Http
+import Json.Decode as JD
+import List.Extra exposing (count, getAt, takeWhileRight)
+import List.Zipper exposing (Zipper, after, current, fromCons, isLast, next, toList, withDefault)
+import Maybe
+import RemoteData
+import SetZipper exposing (RankedTeam, TeamRankings)
+import Setup exposing (Model)
+import String
 
 
 type alias Model =
-    { teams : Zipper (Zipper Team)
+    { teams : Zipper RankedTeam
     , isShowingInfo : Bool
+    , inPage : Int
     }
 
 
-type alias Team =
-    { number : Int
-    , name : String
-    , score : Float
-    }
+type Msg
+    = NextPage
+    | OpenInfo
+
+
+rankingDisplay : RemoteData.RemoteData Http.Error TeamRankings -> TeamRankings
+rankingDisplay rank =
+    case rank of
+        RemoteData.Success validRanking ->
+            validRanking
+
+        _ ->
+            let
+                _ =
+                    Debug.log "didn't succeed"
+            in
+            []
+
+
+createZipper : RemoteData.RemoteData Http.Error TeamRankings -> Zipper RankedTeam
+createZipper rank =
+    fromCons { name = "Open", position = 0 } <| rankingDisplay rank
+
+
+moveZipperBy : Int -> Zipper a -> Zipper a
+moveZipperBy moveBy zipper =
+    if moveBy == 0 || isLast zipper then
+        zipper
+
+    else
+        moveZipperBy (moveBy - 1) <| withDefault (current zipper) <| next zipper
```

Comment:
According to your implementation, if you try to move outside the range you just return the last valid zipper. That's a bad idea which can result in infinite loops easily.
See how [previous](https://package.elm-lang.org/packages/wernerdegroot/listzipper/latest/List-Zipper#previous) and [first](https://package.elm-lang.org/packages/wernerdegroot/listzipper/latest/List-Zipper#first) have the type `Zipper a -> Maybe (Zipper a)`? `x -> Maybe x` means the attempted operation can fail. If you have `x -> x` it means the operation can never fail, because you just return the original `x`, which means a looping operation like `first` might get stuck in an infinite loop - recursing on the first element over and over because it never reaches `Nothing`.

What you probably want here is something like:
```elm
moveZipperBy : Int -> Zipper a -> Maybe (Zipper a)
moveZipperBy n =
    List.foldl ((<<) << Maybe.andThen) Just <| List.repeat n next
```
Explanation: We create a list of `n` `next`s, and then we fold them with composition in order to create a single function that's equivalent to calling `next` `n` times:
```elm
moveZipperBy 3 zipper === next (next (next zipper)) === (next <<' next <<' next) zipper
```
where
```elm
(<<') == (<<) << Maybe.andThen
```
This is a very important idea - we'll talk more about this on Friday 🙂.